provisioner/influx-series.js

```diff
@@ -36,6 +36,15 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
     // Both the bid and price will be the pre-safety factor number
     bid: base.stats.types.Number,
     price: base.stats.types.Number,
+    securityToken: base.stats.types.String,
+  },
+});
+
+// Store when a security token was retreived
+module.exports.securityTokenRetreived = new base.stats.Series({
+  name: 'AwsProvisioner.SecurityTokenRetreived',
+  columns: {
+    securityToken: base.stats.types.String,
```

Comment:
1) We can't effectively make queries over this that tries to measure time between random ids in two different series... We don't even want to write an influxdb query for this. I assume you want to correlation for stats to measure time between submission and running (call back via secret) right?

2) Note, you create the `secret` entity, about the same time as you submit the spot request, so why not simply avoid correlating the instances, and instead add a `created` timestamp to the `secret` entity... Won't that do?